test/ignorehttpserrors.spec.js

```diff
@@ -63,6 +63,7 @@ describeFailsFirefox('ignoreHTTPSErrors', function() {
       expect(securityDetails.subjectName()).toBe('puppeteer-tests');
       expect(securityDetails.validFrom()).toBe(1550084863);
       expect(securityDetails.validTo()).toBe(33086084863);
+      expect(securityDetails.sanList()).toEqual([]);
```

Comment:
Could you add a test with an actual SAN list?

You can use `https://1000-sans.badssl.com/` for the test, and check the length of the resulting array, as well as one particular element (so that the tests show the expected format of a single SAN object).

Reply:
Shouldn't this be included in the local test webserver certificate? Seems risky to me to have the whole test chain depend on network presence & internet presence.

Reply:
Yes, adding some SANs to the local cert would be ideal.

Reply:
@mtoma Do you want to give this a go?

Reply:
@mathiasbynens Yes, it's the plan. I'm just catastrophically busy these days. Any help or guidance with generating the local certificate wit a san list would be greatly appreciated.
Maybe someone knows how the certificate was generated in the first place? openssl command line probably?

Reply:
I have no idea how the existing cert was generated, but it seems we could use something like https://github.com/FiloSottile/mkcert.

Reply:
Done. For future the command line I used to generate the certificate is: 
```
openssl req -x509 -nodes -subj "/CN=puppeteer-tests" -newkey rsa:2048 -keyout key.pem -out cert.pem -addext "subjectAltName=DNS:www.puppeteer-tests.tld,DNS:www.puppeteer-tests-1.tld" -days 3650
```